maquetado/scss/details.scss

```diff
@@ -51,6 +47,11 @@ body {
   }
 }
 
+.book-info {
+  flex-direction: column;
+  width: 25%;
```

Comment:
There is no need to add this here. You can give a fix size to every book thumbnail in your list and then the info will fit the space within your element.